tests/chart_tests/test_houston_configmap.py

```diff
@@ -21,7 +21,7 @@ def common_test_cases(docs):
     prod = yaml.safe_load(doc["data"]["production.yaml"])
 
     assert prod["deployments"]["helm"]["airflow"]["useAstroSecurityManager"] is True
-    assert prod["deployments"]["disableManageClusterScopedResources"] is False
+    assert "disableManageClusterScopedResources" not in prod["deployments"]
```

Comment:
I don't think this test is good enough. The value of `disableManageClusterScopedResources` could be true or false. If we change the configuration in the future and do not update this test, this test may begin to give a false positive.